tests/music_tests.ts

```diff
@@ -0,0 +1,296 @@
+/**
+ * VexFlow - Music API Tests
+ * Copyright Mohit Muthanna 2010 <mohit@muthanna.com>
+ */
+
+declare const QUnit: any;
+declare const ok: any;
+declare const test: any;
+declare const expect: any;
+declare const equal: any;
+
+import { Music } from '../src/music';
+import { KeyManager } from '../src/keymanager';
+
+export const MusicTests = {
```

Comment:
I know that this change is **ugly** but it will help all of us a lot going forward becuase it allows git to detect that the file music_tests.ts` was renamed and not deleted.
Change:
```
export const MusicTests = {
```
to:
``` 
export const MusicTests = (function () {
  return {
```
and add: 
```
})();
```
at the end of the definition after line 296.

At the end of the migration we can remove this unnecessary level, if we want, but during the migration it will allow us to see renamed files decently.

Reply:
@0xfe @wassertim I believe that this one is **very important to facilitate the reviews**. 

I realised that in #872 the diff was present and I did look for the difference. I have also checked in my fork that making the change in this PR fix the issue. 

Look at `music_tests` in: https://github.com/rvilarl/vexflow/pull/13/files

Reply:
Since we have seen this approach has worked on a test file, we don't need it anymore. I reverted back to music_tests.js, we can migrate this one later